StardewArchipelago/Locations/CodeInjections/Modded/SVE/SVECutsceneInjections.cs

```diff
@@ -120,5 +124,27 @@ public static bool UpdateSpecialOrders_StopDeletingSpecialOrders_Prefix()
                 return true; // run original logic
             }
         }
+
+        // Original method runs on SaveLoaded, OnWarped, TimeChanged
+        // private static void FixMonsterSlayerQuest()
+        public static void FixMonsterSlayerQuest_IncludeReleaseofGoals_Postfix()
+        {
+            try
+            {
+                foreach (var voidSpirit in voidSpirits)
+                {
+                    var locationName = $"{MONSTER_ERADICATION_AP_PREFIX}{voidSpirit}";
+                    if (_locationChecker.IsLocationMissingAndExists(locationName))
```

Comment:
This code will not catch all progressive goals

Reply:
Due to the list voidSpirits I imagine?  I've included all related goals now.